crates/cheatcodes/src/env.rs

```diff
@@ -26,6 +26,13 @@ impl Cheatcode for setEnvCall {
     }
 }
 
+impl Cheatcode for envExistsCall {
+    fn apply(&self, _state: &mut Cheatcodes) -> Result {
+        let Self { name } = self;
+        Ok(env::var(name).is_ok().abi_encode())
```

Comment:
Note that `env::var` returns `Err` if it exists but is not valid unicode (see `fn get_env`). This is probably fine since we error on that as well in `getEnv`

Reply:
should I handle this error to still returns true or not then ?